config/initializers/action_mailer.rb

```diff
@@ -9,5 +9,7 @@
   password: ENV['SMTP_USER_NAME'] == 'none' ? nil : ENV['SMTP_PASSWORD'].presence,
   openssl_verify_mode: ENV['SMTP_OPENSSL_VERIFY_MODE'].presence,
   ca_path: ENV['SMTP_OPENSSL_CA_PATH'].presence,
-  ca_file: ENV['SMTP_OPENSSL_CA_FILE'].presence
-}
+  ca_file: ENV['SMTP_OPENSSL_CA_FILE'].presence,
+  read_timeout: ENV['SMTP_READ_TIMEOUT'] || nil,
+  open_timeout: ENV['SMTP_OPEN_TIMEOUT'] || nil
```

Comment:
Aren't these supposed to be integer values?
```suggestion
  read_timeout: ENV['SMTP_READ_TIMEOUT']&.to_i,
  open_timeout: ENV['SMTP_OPEN_TIMEOUT']&.to_i,
```

Reply:
Thank you @knu
I just assumed these would work the same as the SMTP_PORT one, but I guess action mailer is doing that conversion internally for the port.